charts/artifact-registry/chart/scripts/install_helm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
We want to fail if any of the commands fail.
```suggestion
#!/bin/bash
set -e
```